src/main/kotlin/no/nav/syfo/infrastructure/database/repository/VurderingRepository.kt

```diff
@@ -35,12 +35,25 @@ class VurderingRepository(private val database: DatabaseInterface) : IVurderingR
                 is ManglendeMedvirkningVurdering.Forhandsvarsel -> connection.saveVarsel(pVurdering.id, vurdering.varsel)
                 else -> null
             }
-            val pVurderingPdf = connection.saveVurderingPdf(pVurdering.id, pVurdering.createdAt, vurderingPdf)
+            connection.saveVurderingPdf(pVurdering.id, pVurdering.createdAt, vurderingPdf)
             connection.commit()
 
             pVurdering.toManglendeMedvirkningVurdering(pVarsel)
         }
 
+    override fun getVurderinger(personident: Personident) =
+        database.connection.use { connection ->
+            connection.prepareStatement(GET_VURDERING).use {
+                it.setString(1, personident.value)
+                it.executeQuery().toList {
+                    toPVurdering()
+                }
+            }.map { pVurdering ->
+                val pVarsel = connection.getVarselForVurdering(pVurdering)
+                pVurdering.toManglendeMedvirkningVurdering(pVarsel)
+            }
+        }
```

Comment:
Burde vi her bare hente vurderingen i sin helhet i en query? 😊 Kommer sikkert ikke til å bli tilfelle, men om en person har mange vurderinger, så kan denne connection'en leve unødvendig lenge og ta unødvendig lang tid.

Reply:
Ganske sikker på at dette går veldig bra slik det står: begge query'ene er på indekserte kolonner.

Reply:
Tror ikke det er krise for denne appen, men det er typisk disse spørringene vi oppdaget at tok lang tid før sommeren i `syfooversiktsrv` når man møtte gjøre det for mange personer i en `map()`. Det går også an å følge med på i Grafana 👍🏼